publictask/backend/plugin/src/main/resources/config/liquibase/changelog/add-public-task-plugin-entity.xml

```diff
@@ -27,6 +27,9 @@
             <column name="user_task_id" type="${uuidType}">
                 <constraints nullable="false"/>
             </column>
+            <column name="process_business_key" type="VARCHAR(30)">
```

Comment:
voor nu gaat dit goed omdat alleen jij deze db hebt maar voortaan moet je even een nieuwe change set maken in deze file om een propertie toe te voegen aan de table. anders gaat liquibase stuk

Reply:
check. thanks!